config.json

```diff
@@ -20,8 +20,7 @@
   },
   "files": {
     "solution": [
-      "src/lib.cairo",
-      "Scarb.toml"
```

Comment:
@ErikSchierboom this should remove _Scarb.toml_ from the online editor?
http://forum.exercism.org/t/need-maintainers-to-test-the-new-cairo-track/12154/10

Reply:
@misicnenad Yes, it will remove it from the online editor. What I would suggest you do is to not remove it, but instead put it in the `invalidator` key. This will ensure that if the `Scarb.toml` file changes, students will be prompted to update the exercise. An invalidator files are not shown in the online editor. See https://exercism.org/docs/building/tracks/practice-exercises#h-file-meta-config-json

Reply:
Done https://github.com/exercism/cairo/pull/99/commits/160a33a400548d41931165b2c3c49780084d9b29